Added links from Flytelab (#652)
flyteorg/flytesnacks: cookbook/docs/tutorials.rst

       :text: 🧪 Bioinformatics
       :classes: btn-block stretched-link
    ^^^^^^^^^^^^

>     Perform computational biology with Flyte.

Reviewer: @SmritiSatyanV let's create a new "Flytelab" section header below tutorials with a separate set of panels for flytelab projects. This is so we differentiate between "flytesnacks" examples and "flytelab" repo projects,

    ---

    .. link-button:: flytelab
       :type: ref

>        :text: 🧪 Flytelab

Reviewer 2: ```suggestion
       :text: 🔬 Flytelab
```

       :text: 🧪 Bioinformatics
       :classes: btn-block stretched-link
    ^^^^^^^^^^^^
    Perform computational biology with Flyte.

*********
Flytelab
*********

> This repository contains Machine Learning projects that use the Flytekit Python SDK to implement end-to-end ML applications.

Reviewer 2: ```suggestion
This section contains end-to-end ML projects using Flyte.
```
